src/libs/ck-libs/NDMeshStreamer/NDMeshStreamer.h

```diff
@@ -24,6 +28,12 @@
 
 #define TRAM_BROADCAST (-100)
 
+//#define TRAM_BUFFER_SIZE 20480
+
+//#define TRAM_SEND_CUTOFF_NUMERATOR 5
+//#define TRAM_SEND_CUTOFF_DENOMINATOR 10
+//#define TRAM_SEND_FRACTION_NUMERATOR 5
+//#define TRAM_SEND_FRACTION_DENOMINATOR 10
```

Comment:
Are these needed anymore? If not, removing them would be better.

Reply:
Removed.